src/translations/en/index.js

```diff
@@ -165,8 +165,18 @@ const translations = {
     createAccount: 'Create account',
     moreOptions: {
       edit: 'Edit Account',
+      icns: 'Add ICNS Domain',
       copy: 'Copy Address',
     },
+    icns: {
+      setICNS: 'Choose ICNS Name',
+      learnMore: 'Learn More',
+      emptyState:
+        'We weren’t able to find any ICNS names in your Plug account. ',
```

Comment:
do we need that space at the end? same for `info`

Reply:
Yes!